server/handlers/requestHandlers.go

```diff
@@ -58,10 +68,73 @@ func (h *Handlers) LoginRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	expirationTime := time.Now().Add(time.Second * 20)
+
+	claims := &Claims{
+		Username: login.Email,
+		StandardClaims: jwt.StandardClaims{
+			ExpiresAt: expirationTime.Unix(),
+		},
+	}
+
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, claims)
+	tokenString, err := token.SignedString(jwtKey)
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	http.SetCookie(w,
+		&http.Cookie{
+			Name:    "token",
+			Value:   tokenString,
+			Expires: expirationTime,
+		})
+
 	log.Println("INFO [handlers/requestHandlers.go] Successful Log In")
 	http.Redirect(w, r, "/api/login_request_success", http.StatusSeeOther)
```

Comment:
This was actually a part of Ekaterina's MR but I didn't notice. Can you change this line to h.TokenValid(w, r)? I don't think we need to go through the router since the function is part of the same class

Reply:
Wasn't sure how I suppose to change it, I tried h.tokenValid(w,r) as well as h.LoginRequestSuccess(w,r) and they both gave me bug